### PR TITLE
docs/esp8266/tutorial: esptool 1.3 added Python 3 support.

### PR DESCRIPTION
**Pull request by JYOuyang**
esptool 1.3 now supports both Python 2.7 and 3.4+.
Updated github link to now-official espressif repo.

### CODE REVIEW

**Comment by dpgeorge** [comment]
Yes I'm now using this version and it works with Python 3.

But if people are still using the old version of esptool then they will have problems with Python 3.  The old esptool still works just fine so maybe it's worth adding a note in parenthesis that older esptools work but require Python 2.7

**Comment by JYOuyang** [comment]
Sure. I've added a paragraph for this (and combined it with the existing 1.2.1 blurb).

File: docs/esp8266/tutorial/intro.rst [hunk 1]
```diff
@@ -72,23 +72,25 @@ For best results it is recommended to first erase the entire flash of your
...
+using pip::
 
     pip install esptool
 
-It requires Python 2.7, so you may need to use ``pip2`` instead of ``pip`` in
-the command above.  Any other
-flashing program should work, so feel free to try them out, or refer to the
-documentation for your board to see its recommendations.
+Versions starting with 1.3 support both Python 2.7 and Python 3.4 (or newer).
+Any older version (at least 1.2.1 is needed) works fine but will require Python
```
Comment (pfalcon) [inline docs/esp8266/tutorial/intro.rst]: Apparently, not "any older", but "an older".

Comment (JYOuyang) [inline docs/esp8266/tutorial/intro.rst]: I agree. Would be a bit disconcerting to say any version followed immediately with the 'at least 1.2.1' requirement.

**Comment by dpgeorge** [comment]
Thanks.  This was squashed and merged in e73a0b944fbb391f314e7f64ec49e503d53d179a.

**Pull request closed without merging**